python/example_code/lookoutvision/inference.py

```diff
@@ -95,50 +83,204 @@ def download_from_s3(s3_resource, photo):
 
         return local_file
 
+    @staticmethod
+    def reject_on_classification(image, prediction, confidence_limit):
+        """
+        Returns True if the anomaly confidence is >= than the supplied confidence limit.
```

Comment:
This reads weird to me. I'd write out '>=' so: "confidence is greater than or equal to the supplied confidence..."

Reply:
Fixed.